js/src/tooltip.js

```diff
@@ -435,6 +442,10 @@ class Tooltip {
     return title
   }
 
+  getAdditionalClasses() {
```

Comment:
TBH I don't know if we need the separate function. And if we did, it should begin with an underscore since it's private, and moved in the private block

Reply:
yep it should be a private method and start with an `_`
btw it should return an array of classes and with we would be able to do just one call to `addClass` it should be better in terms of perfs

Reply:
Changed name to `_getCustomClass()`. See below for discussion about using built-in array processing in `$.addClass`.